playbooks/group_vars/benchmark_hosts/00-defaults.yml

```diff
@@ -69,7 +69,7 @@ fio_bench_list_default:
     rw: write
     iodepth: 8
     numjobs: 1
-    run_bench: "{{ ebs_rbd_bench | default(true) }}"
+    run_bench: "{{ ebs_rbd_bench | default(false) }}"
```

Comment:
If we define these variables once in this file we will not have to give them defaults in multiple places.

I think we should set at the top of this file:
```
ebs_rbd_bench: false
```

And then set
```
    run_bench: "{{ ebs_rbd_bench }}"
```

We should do this for all the run_bench variables so we are not giving them default values in multiple playbooks.

Reply:
addressing comments in next pr